File: patterns/javascript-patterns/pattern78.js
// Pattern 78: ...

// *       *
//  *     *
//   *   *
//    * *
//     *

const n = 5;

let px = 1;
let py = n * 2 - 1;

for (let i = 1; i <= n; i++) {
    let row = '';

    for (let j = 1; j <= n * 2; j++) {
        if (j === px || j === py) {
            row += '*';
        } else {
            row += ' ';
        }
    }

    px++;
    py--;
    console.log(row);
}

File: patterns/python-patterns/pattern78.py
# Pattern 78: ...

# *       *
#  *     *
#   *   *
#    * *
#     *

n = 5
px = 1
py = n * 2 - 1

for i in range(1, n + 1):
    for j in range(1, n * 2 + 1):
        if j == px or j == py:
            print("*", end="")
        else:
            print(" ", end="")
    px += 1
    py -= 1
    print()

